downstream/assemblies/platform/assembly-platform-install-scenario.adoc

```diff
@@ -22,10 +22,32 @@ The {PlatformNameShort} installer allows you to deploy *only one* {HubName} per
 Installer does not require a user to be logged in as root to run `./setup.sh`. The user will need to properly configure the environment variable `ANSIBLE_BECOME_METHOD` for the preferred method of privilege escalation to root. The default method is `sudo`.
 ====
 
-This installation option includes two supported scenarios:
+.Prerequisites
+
+* You chose and obtained a platform installer from the https://access.redhat.com/downloads/content/480/ver=2.1/rhel---8/2.1/x86_64/product-software[Red Hat Ansible Automation Platform Product Software].
+* You are installing on a machine that meets base system requirements.
+* You have created a Red Hat Registry Service Account, following the instructions in the https://access.redhat.com/RegistryAuthentication#creating-registry-service-accounts-6[Creating Registry Service Accounts guide].
+
+There are a number of supported installation scenarios for {PlatformName}. To install {PlatformName}, you must edit the inventory file parameters to specify your installation scenario using one of the examples below:
```

Comment:
To avoid fluff and to hopefully add clarity, what about this:

"You can choose among a few supported scenarios for installing {PlatformName}. You must edit the inventory file parameters to specify your installation scenario. Choose one of the following inventory file examples [to begin?]"